src/styles/components/_tabs.scss

```diff
@@ -28,29 +23,27 @@
 
   // NOTE: Never hide the scrollbar except under very specific situations.
   // Reach out to UX team before hiding a scrollbar.
+  /* stylelint-disable-next-line property-disallowed-list */
   scrollbar-width: none; // Hide scrollbar in firefox
 
-  &::-webkit-scrollbar
-  {
+  &::-webkit-scrollbar {
     display: none; // Hide scrollbar in Safari and Chrome
   }
 
-  &::after
-  {
+  &::after {
     position: sticky;
     right: 0;
     display: block;
     width: space(xxs);
     height: space(6x);
-    box-shadow: - space(xxs) 0 space(md) space(xs) rgba($neutral-00, 0.2);
+    box-shadow: calc(0rem - space(xxs)) 0 space(md) space(xs) rgba($neutral-00, 0.2);
```

Comment:
intentional?

Reply:
Stylelint was giving an error because it was getting prettiered to `-space(xxs)` and stylint was complaining it didn't recognize that function